std/string.d

```diff
@@ -193,12 +193,16 @@ public import std.uni : icmp, toLower, toLowerInPlace, toUpper, toUpperInPlace;
 public import std.format : format, sformat;
 import std.typecons : Flag;
 
-import std.meta;
-import std.range.primitives;
-import std.traits;
+import std.meta; // AliasSeq, staticIndexOf
```

Comment:
If you know what you need from std.meta why don't you actually only import that. Having them as comments behind the actual import is IMO strange.